application/F3DOptionsTools.cxx

```diff
@@ -192,16 +192,16 @@ void PrintHelpPair(
   std::string_view key, std::string_view help, int keyWidth = 10, int helpWidth = 70)
 {
   std::stringstream ss;
-  ss << "  " << std::left << std::setw(keyWidth) << key << " " << std::setw(helpWidth) << help;
+  ss << "  " << std::left << std::setw(keyWidth) << key << " \n" << std::setw(helpWidth) << help;
```

Comment:
this is not what I meant. I meanty to add a new argments to PrintHelpPair so this char here can be controlled.

Reply:
or add the `\n` automatically if the key is longer that the padding maybe?
```cpp
  ss << "  " << std::left << std::setw(keyWidth) << key;
  if(key.size() > keyWidth){
   ss << "\n  " << std::setw(keyWidth) << " ";
  }
  ss << " " << std::setw(helpWidth) << help;
```
```
Examples:
  f3d file.vtu -xtgans                               View a unstructured mesh in a typical nice looking sciviz style       
  f3d file.glb -tuqap --hdri-file=file.hdr --hdri-ambient --hdri-skybox
                                                     View a gltf file in a realistic environment                           
  f3d file.ply -so --point-size=0 --comp=-2          View a point cloud file with direct scalars rendering                 
  f3d folder                                         View all files in folder       
```

Reply:
It should be fine rn.